packages/web/src/styles/AIAnswer.js

```diff
@@ -363,6 +363,7 @@ export const MessageInput = styled(TextArea)`
 	border: 1px solid #ccc;
 	color: ${props => props.theme.colors.textColor};
 	height: 42px;
+	padding-left: 35px;
```

Comment:
@SavvyShah What is this for?

Reply:
[Similar change done for vue](https://github.com/appbaseio/reactivesearch/commit/0a99ea7ab4cf49ccea6771d3d3320df50fc55cb8).